src/actinia_core/core/stac_importer_interface.py

```diff
@@ -142,26 +144,69 @@ def _stac_import(self, stac_collection_id=None, semantic_label=None,
 
             stac_processes = []
 
+            # Create the strds
+
+            # datetime object containing current date and time
+            now = datetime.now()
+
+            # dd/mm/YY H:M:S
+            dt_string = now.strftime("%d%m%Y%H%M%S")
+
+            t_name = f'{stac_name}{dt_string}'
+
+            exec_params = ["type=strds",
+                           "temporaltype=absolute",
+                           "output=%s" % t_name,
+                           "title=%s" % stac_name,
+                           "description=%s" % stac_collection_id
+                           ]
+
+            p = Process(
+                exec_type="grass",
+                executable="t.create",
+                executable_params=exec_params,
+                id=f"t_create_{os.path.basename(stac_name)}",
+                skip_permission_check=True
+            )
+
+            stac_processes.append(p)
+
             for key, value in stac_result.items():
 
-                for name_id, url in value.items():
+                for name_id, url, item_date in value.items():
 
                     output_name = stac_name + "_" + key + "_" + name_id
 
                     # From Here Onwards, the Process build starts
                     exec_params = ["input=%s" % "/vsicurl/"+url,
                                    "output=%s" % output_name,
-                                   "-o"]
+                                   "extent=region"]
```

Comment:
Should also `resolution=region` be used? At time, an estimated resolution value is used.

Reply:
Yes, I didn't know which were more convenient in this case. But it makes sense to me, to set resolution as regional 👍🏽

Reply:
I tend to use the estimated resolution because e.g. with Sentinel-2 the different bands have different native resolutions. Resampling all bands to the same resolution might not be desired.

Reply:
This resolution refers to Spatial resolution, right? If true, resampling the resolution might be convenient in terms of operational processing inside the Location. However, it can cause a loss of information if the image is upscaling.